pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "slacktools"
```

Comment:
Just FYI, this [name](https://pypi.org/project/slacktools/) is already taken in case we ever want to publish it

Reply:
any suggestions?